src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -26,8 +26,18 @@
 
     // Conductor properties
     private String[] currentSchemes;
-    private int timeout = 5;
-    private int retries = 5;
+    /***
+     * @deprecated in favor of {@link #appiumRequestTimeout} since 0.19.1 for clarity sake on what the variable actually is used for
+     */
+    @Deprecated
+    private int timeout = 5; // deprecated in favor of `appiumRequestTimeout`
+    private int appiumRequestTimeout = 5;
+    /***
+     * @deprecated in favor of {@link #implicitWaitTime} since 0.19.1 for clarity sake on what the variable actually is used for
+     */
+    @Deprecated//(since="0.19.1")
+    private int retries = 5; // deprecated in favor of `implicitWaitTime`
```

Comment:
Maybe get rid of the old variable and have the getter/setter use the new one. Then have the yaml parsing set the new variable from the old one if it's there.

A nice thing about doing that logic in the yaml parsing is you can print out a warning that the field is deprecated. Otherwise if you just used the yaml you wouldn't know.

Reply:
Addressed in 39bfe516f6761db0579cd8c45c80f43e0a98f75c
- Parsing the properties in `ConductorConfig.java:readProperties()`, we now check for the deprecated variables. If we find one as a property, we instead set the new variable and spit out a system.out to tell the user to stop using the deprecated variable.

Reply:
Also, I opted against adding a yaml that had the old variables in them and creating a test for that. If you think it should be in there, I can definitely go about doing that.

(FWIW I did test this worked by manually updating `android_defaults.yaml`)